xml/obs/openSUSE:Factory:zSystems.xml

```diff
@@ -4,7 +4,7 @@
     distri="opensuse">
     <flavor name="Tumbleweed-DVD" folder="*product*" iso="1" archs="s390x">
         <repos>
-            <oss folder="openSUSE-ftp-ftp" debug="{java*,kernel-default-debug*,kernel-default-base-debug*,mraa-debug*}" source="{coreutils*,yast2-network*}" mirror="1" dest="openSUSE-Tumbleweed-oss-$arch"/>
+            <oss folder="openSUSE-ftp-ftp" debug="{java*,kernel-default-debug*,kernel-default-base-debug*,mraa-debug*}" source="{coreutils*,yast2-network*}" mirror="1" dest="Tumbleweed-oss-$arch"/>
```

Comment:
for the future: I think it's a good idea to put something like this into here as a comment. I hope that's also possible with this XML file format.